src/website/app/demos/Link/examples/reactRouter.js

```diff
@@ -27,7 +27,9 @@ import {
 export default {
   id: 'react-router',
   title: 'ReactRouter',
-  description: `This example passes a ReactRouter Link to a Mineral UI Link's element prop in order to apply Mineral UI styles and achieve client side routing.`,
+  description: `Client-side routing can be achieved by passing a [ReactRouter Link](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md) to the \`element\` prop.
```

Comment:
This makes it sound like we only support ReactRouter, but I don't think that's true. I like that you linked their Link.

Reply:
I generalized the example to "Routing"